Plater.lua

```diff
@@ -4551,19 +4551,19 @@ end
 	end
 	
 	--internal function to change the health bar color
-	function Plater.ChangeHealthBarColor_Internal (healthBar, r, g, b, forceNoLerp) --private
-		if (r ~= healthBar.R or g ~= healthBar.G or b ~= healthBar.B) then
-			healthBar.R, healthBar.G, healthBar.B = r, g, b
+	function Plater.ChangeHealthBarColor_Internal (healthBar, r, g, b, a, forceNoLerp) --private

```

Comment:
need to keep an eye on this: adding a nil-check for alpha here might be useful as well to counter potential issues with profiles without the alpha value in the colors. this should not happen, as the default values are adjusted and the options store the alpha as well, but one never knows...